schema_enforcer/instances/file.py

```diff
@@ -34,22 +34,31 @@ def __init__(self, config):
 
         # For each instance file, check if there is a static mapping defined in the config
         # Create the InstanceFile object and save it
-        for root, filename in files:
-            matches = []
+        for root, filename in instance_files:
+            matches = set()
             if filename in config.schema_mapping:
-                matches.extend(config.schema_mapping[filename])
+                matches.update(set(config.schema_mapping[filename]))
```

Comment:
```suggestion
                matches.add(config.schema_mapping[filename])
```

Reply:
The value of `schema_mapping[filename]` is a list here, so I believe we have to use "update" as it isn't an individual element?